treescript/src/treescript/gecko/mercurial.py

```diff
@@ -137,7 +137,7 @@ async def validate_robustcheckout_works(config):
 
 
 # checkout_repo {{{1
-async def checkout_repo(config, task, repo_path):
+async def checkout_repo(config, task, source_repo, repo_path):
```

Comment:
nit: consider pulling this refactor out into a separate commit

Reply:
Sorry, I should have done that originally. Is it worth doing retroactively?